src/components/GeoListInput/index.js

```diff
@@ -107,62 +125,89 @@ export default class GeoListInput extends React.PureComponent {
     }
 
     valueLabelSelector = (v) => {
-        const option = this.geoOptionsById[v];
+        const option = this.geoOptionsById[v.key];
         if (this.props.regions.length > 0) {
             return `${option.regionTitle} / ${option.label}`;
         }
         return option.label;
     }
-    valueKeySelector = v => v;
+    valueKeySelector = v => v.key;
```

Comment:
Why no static?

Reply:
I actually feel really uncomfortable that `valueLabelSelector` is not static while `valueKeySelector` is. I like them to be together.

`static` must be used when you want to use the function without instantiating the class or when the caller expects the function not to use `this` at all. But the selector functions have no such constraints. Normally we would make them static anyway in hope of keeping pieces of code organized.

However, there is not much advantage in using `static` here, since the two related methods will only get separated.